images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -140,6 +201,22 @@ meteringconfig_create_presto_aws_credentials: "{{ _presto_spec.config.createAwsC
 meteringconfig_create_presto_tls_secrets: "{{ _presto_spec.presto.config.tls.enabled and _presto_spec.presto.config.tls.createSecret | default(false) }}"
 meteringconfig_create_presto_auth_secrets: "{{ _presto_spec.presto.config.auth.enabled and _presto_spec.presto.config.auth.createSecret | default(false) }}"
 
+#
+# Enabling Presto TLS/Auth by Default
+#
+meteringconfig_ansible_generate_presto_tls_secret: "{{ meteringconfig_spec.tls.enabled and _presto_spec.presto.config.tls.enabled and not _presto_spec.presto.config.tls.createSecret and _presto_spec.presto.config.tls.secretName == '' | default(false) }}"
+meteringconfig_ansible_generate_presto_auth_secret: "{{ meteringconfig_spec.tls.enabled and _presto_spec.presto.config.auth.enabled and not _presto_spec.presto.config.auth.createSecret and _presto_spec.presto.config.auth.secretName == '' | default(false) }}"
+meteringconfig_ansible_generate_presto_secrets: "{{ meteringconfig_ansible_generate_presto_tls_secret or meteringconfig_ansible_generate_presto_auth_secret }}"
+meteringconfig_ansible_generate_presto_tls_secret_name: "{{ presto-server-tls if meteringconfig_ansible_generate_presto_tls_secret else _presto_spec.presto.config.tls.secretName }}"
```

Comment:
Are these variables needed since we set them in https://github.com/operator-framework/operator-metering/pull/769/files#diff-0e59c4148c558b24a0bdbbd4229bccd6R134 above?